### PR TITLE
Fixes test with log file and adds skip option

### PR DESCRIPTION
**Pull request by Cyb3r-Jak3**
Biggest Changes
---
- Fixes the test with log file and separate normal test from config test
- Adds Skip file option. Similar to the blacklist option.

### CODE REVIEW

**Comment by Cyb3r-Jak3** [comment]
Pypy implementation in Github Actions does not appear to be working. See https://github.com/actions/virtual-environments/issues/1716

**Comment by Cyb3r-Jak3** [comment]
@svenkreiss Would you be open to switching to GitHub Actions for the CI?

I have been messing around with it a bit and if switching there is a lot more tests that can be run such as different os and java versions. Here is an example of what it would look like: https://github.com/Cyb3r-Jak3/html5validator/actions/runs/285811420.
That is testing the same python versions, with java 8-13 and ubuntu 16, 18, and 20, mac-os and windows. Allows for seeing if there are issues with different java version or OS.

**Comment by Cyb3r-Jak3** [comment]
@svenkreiss I added a new option `--skip` to allow users to skip certain files. This was requested in the GitHub action. https://github.com/Cyb3r-Jak3/html5validator-action/issues/12.

Was not sure if you want a separate PR for it.

**Comment by khusika** [comment]
Is there any plan to merge this?

**Comment by svenkreiss** [comment]
Thanks @Cyb3r-Jak3 for the PR and @khusika for the reminder.

The moved tests are a blocker for me. I want to be able to run these tests without the full source tree.

I am also against a new command line argument but it does make a lot of sense to extend `--blacklist` to also remove individual files and not only directories.

**Comment by Cyb3r-Jak3** [comment]
@svenkreiss 
Tests are back to where they were. Also, make it so the blacklist command does files and directories.

You are definitely going to want squash merge this as the config tests don't work on Windows due to the path.

File: setup.py [hunk 1]
```diff
@@ -18,14 +18,13 @@
 setup(
     name='html5validator',
     version=__version__,
-    packages=['html5validator', 'html5validator.tests', 'vnujar'],
```
Comment (svenkreiss) [inline setup.py:21]: revert

File: setup.py [hunk 2]
```diff
@@ -18,14 +18,13 @@
...
+    packages=['html5validator', 'vnujar'],
     license='MIT',
     description='Validate HTML5 files.',
     long_description=open('README.rst').read(),
     author='Sven Kreiss',
     author_email='me@svenkreiss.com',
     url='https://github.com/svenkreiss/html5validator',
-
```
Comment (svenkreiss) [inline setup.py:28]: revert

File: html5validator/validator.py [hunk 3]
```diff
@@ -48,12 +49,12 @@ def all_files(directory='.', match='*.html', blacklist=None,
         for b in blacklist:
             if b in dirnames:
                 dirnames.remove(b)
+            if b in filenames:
+                filenames.remove(b)
 
         if skip_invisible:
             # filter out directory names starting with '.'
-            invisible_dirs = [d for d in dirnames if d[0] == '.']
```
Comment (svenkreiss) [inline html5validator/validator.py:54]: revert

File: html5validator/tests/test_config.py [hunk 4]
```diff
@@ -1,173 +1,187 @@
-# -*- coding: utf-8 -*-
```
Comment (svenkreiss) [inline html5validator/tests/test_config.py:1]: Why is this entire file different?

Same for all the files in `tests/config_files`?

File: MANIFEST.in [hunk 5]
```diff
@@ -1,3 +1,2 @@
 include README.rst LICENSE vnujar/vnu.jar
-recursive-include tests *
-recursive-exclude tests *.pyc .DS_Store
+prune html5validator/tests
```
Comment (svenkreiss) [inline MANIFEST.in:2]: remove

**Review by svenkreiss — COMMENTED** [review]
Thanks for fixing those. 

The diff is still huge and it is not clear to me why. Can you revert so that the diff becomes readable?

I have added some further requests to revert changes that I think were from your previous changes and are now unnecessary. Maybe I am wrong about some of them, so please reply with a short comment why those are necessary now. Thanks!

**Comment by Cyb3r-Jak3** [comment]
Looking through everything, I agree this got incredibly messy. I am going to close out this PR and start fresh.

**Pull request closed without merging**